cmd/earth/main.go

```diff
@@ -605,7 +636,20 @@ func (app *earthApp) run(ctx context.Context, args []string) int {
 }
 
 func (app *earthApp) actionBootstrap(c *cli.Context) error {
-	return app.insertBashCompleteEntry()
+
+	err := app.insertBashCompleteEntry()
+	if err != nil {
+		return err
+	}
+
+	err = app.insertZSHCompleteEntry()
+	if err != nil {
+		return err
+	}
+
+	fmt.Fprintf(os.Stderr, "Bootstrapping successful; you may have to restart your shell for changes to take effect (e.g. run \"exec $SHELL\")\n")
```

Comment:
```suggestion
	fmt.Fprintf(os.Stderr, "Bootstrapping successful; you may have to restart your shell for autocomplete to get initialized (e.g. run \"exec $SHELL\")\n")
```

Being a little more specific about which changes would take effect.